core/client/src/main/java/alluxio/client/block/LocalBlockInStream.java

```diff
@@ -89,14 +90,12 @@ public void close() throws IOException {
       }
       mBlockWorkerClient.unlockBlock(mBlockId);
     } finally {
-      mContext.releaseWorkerClient(mBlockWorkerClient);
+      mClosed = true;
```

Comment:
I don't think it  matter much. Maybe slightly better to consider it as closed if close throws an exception. I moved this mainly because I want to keep it consistent with other files such as LocalBlockOutStream.java.